content/cloud/labels.md

```diff
@@ -0,0 +1,65 @@
+title = "Labels"
+template = "cloud_main"
+date = "2023-10-30T00:22:56Z"
+enable_shortcodes = true
+[extra]
+url = "https://github.com/fermyon/developer/blob/main/content/cloud/labels.md"
+
+---
+- [Labels Overview](#labels-overview)
+- [Benefits of labels](#benefits-of-labels)
+- [How To Create A Label](#how-to-create-a-label)
+- [How to Delete A Label](#how-to-delete-a-label)
+- [Next Steps](#next-steps) 
+
+# Labels Overview
+
+Spin applications are inherently ephermal resources; therefore, you don't necessarily want other Fermyon Cloud resources (such as NoOps SQL Databases) to share the exact same lifecycle. Furthermore, as you build and prototype your Spin application you may want to connect to other resources without necessarily having to recompile your application, but you still need some construct to explain the relationship between your Spin application and the other resource you need it to connect with. 
+
+This is where Fermyon Cloud `labels` are useful. `labels` are construct that allow developers to establish a relationship between a Spin application and it's dependent Fermyon Cloud resources while still preserving independent lifecycles for the respective resources. `Labels` are application scoped and an indirect reference to a named Fermyon Cloud resource that belongs to the user. Whenever that Spin application needs to access that resource, it will use the `label` as reference. The Spin application does not need to know the true name of the resource it's interacting with. This also means a `label` is well understood by all the components within it's respective application, but it doesn't hold any significance to other applications. 
+
+> At this time, only NoOps SQL DB supports `labels`. However, in the future other Fermyon Cloud resources such as Key Value Store will support `labels` as well. 
+
+## Benefits of Labels
+
+Benefits of the Fermyon Clould `labels` include:
+
+* **Easy Sharing**: Share your Fermyon Cloud resource across applications effortlessly.
+* **Resource Creation Control**: You decide when to create and delete Fermyon Cloud resources.
```

Comment:
Again, I feel like this page is presenting things the wrong way round. It starts from labels and goes to "labels allow you to decide" when actually what happens is "you need to be able to decide" and labels are how we solve that.